fuzz/README.md

```diff
@@ -20,6 +20,29 @@ On Nix, you can obtain these libraries by running
 
 and then run fuzz.sh as above.
 
+# Fuzzing with weak cryptography
+
+You may wish to replace the hashing and signing code with broken crypto,
+which will be faster and enable the fuzzer to do other-wise impossible
```

Comment:
> other-wise

Never seen it written like this, always "otherwise", is this correct?

Reply:
I'm not sure what I was thinking here :). Yeah, the dash shouldn't be there.

Reply:
ROFL, a whole bunch of times over the last few years when reading your writing @apoelstra I've thought "damn this guy knows his English", this slip up "other-wise" made me hold my stomach laughing - so glad its not only me that does shit like this :)